mindsdb/integrations/utilities/handler_utils.py

```diff
@@ -49,6 +49,6 @@ def get_api_key(
 
     if strict:
         raise Exception(
-            'Missing API key "api_key". Either re-create this ML_ENGINE specifying the `api_key` parameter, or re-create this model and pass the API key with `USING` syntax.'
+            "Missing API key f'"{api_name.lower()}_api_key"'. Either re-create this ML_ENGINE specifying the f'"{api_name.lower()}_api_key"' parameter, or re-create this model and pass the API key with `USING` syntax."
```

Comment:
The `f` that indicates this is an f-string should be present only once, at the start of the string. Like so:

```python
f"Missing API key '{api_name.lower()}_api_key'. Either re-create this ML_ENGINE specifying the '{api_name.lower()}_api_key' parameter, or re-create this model and pass the API key with `USING` syntax."
```

Reply:
(as it is right now, it will not work)

Reply:
That's right. Thank you @paxcema